stdlib/public/core/CTypes.swift

```diff
@@ -62,7 +62,12 @@ public typealias CFloat = Float
 /// The C 'double' type.
 public typealias CDouble = Double
 
-// FIXME: long double
+/// The C 'long double' type.
+#if arch(x86_64) || arch(i386)
+public typealias CLongDouble = Float80
+#else
+public typealias CLongDouble = Double
```

Comment:
Should this be more explicit to rule out architectures we haven't thought about? Technically there's support for pp64le in tree.

Reply:
Yeah, I think ppc64le uses a 128b long double, as does non-Darwin AArch64, IIRC.

Reply:
I'm not saying you need to implement those now, but the typealias shouldn't lie.